receiver/elasticsearchreceiver/metadata.yaml

```
@@ -750,3 +760,22 @@ metrics:
       value_type: int
     attributes: [ ]
     enabled: true
+  # these metrics are from /*/_stats and are index level metrics
+  elasticsearch.index.operations.completed:
```

```suggestion
  elasticsearch.index.operations.count:
```

Typically we use `.count` for metrics like this. If "completed" is not implied, it should probably be an attribute, and we should also find the value for "failed" or similar.

---

I picked this name, because this is how an analogous metric is called for the nodes:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6dc71bd739b4eaa407a9a596dc57ca4884e6313e/receiver/elasticsearchreceiver/metadata.yaml#L297

More context:
Some of these statistics for operations have only total number of operations, some also have values like `current` which are assumably counts of undergoing operations.  As for now, this receiver scrapes only `total` values. The metrics I have added in this PR also contain only `total` values and in the nearest future I'll be adding more metrics, but these also will be only `total` values. 
So, for now it seems reasonable to skip this attribute. However, if someone wanted to add metric like number of current merges, they would have to add an attribute, which would be a breaking change. I guess that the safest option would be to add this attribute now - the only downside is that it won't be consistent with how it's done for the nodes.

---

I don't think it would be appropriate to add an attribute to a value that already represents a total. 

Let's leave it as you originally proposed, based on consistency with the node metric.